homer0/projext-plugin-webpack
# Implement "opener" for the dev server

### What does this PR do?

If you were to disable the automatic reload, the webpack dev server would open the browser but using a URL like `/webpack-dev-server/...`, so instead of opening the browser using the options of the dev server, I added `opener`.

### How should it be tested manually?

Disable automatic reload and run the web server, it should open on the root.

And of course...

```bash
npm test
# or
yarn test
```
